### ISSUE TITLE
[Feature] Including build failure log in main output

### ISSUE THREAD

**Issue by KishanBagaria** [yarnpkg/berry#3349]
We're building native deps on GitHub Actions. When build failures occur, yarn saves the log to a build.log file in a random location. Only way for us to see the log is to upload `C:\Users\RUNNER~1\AppData\Local\Temp\xfs-*\build.log` as CI artifacts.

```
  ➤ YN0009: │ foo@https://github.com/bar/foo.git#commit=hash couldn't be built successfully (exit code 1, logs can be found here: C:\Users\RUNNER~1\AppData\Local\Temp\xfs-150f75fd\build.log)
515
```

Is it possible to include build.log in the main log (process.stdout) instead of saving to a new file?

Edit: overlooked the logs at the top

**Comment by castarco** [comment]
How was this problem fixed? It's still a thing today (I can't find any option to configure where the logs are placed, at least that would allow us to upload it as an artifact for later inspection, but the current randomised path makes it almost impossible).

**Comment by Robula** [comment]
@KishanBagaria Would you mind answering castarco's question so that users looking for this feature know where to find it, please?

**Comment by clemyan** [comment]
In CI environments, the `enableInlineBuilds` config defaults to `true`, which means ALL builds outputs (not only failing ones) are written to stdout with a prefix

**Comment by Robula** [comment]
> In CI environments, the `enableInlineBuilds` config defaults to `true`, which means ALL builds outputs (not only failing ones) are written to stdout with a prefix

It's never worked in Google Cloud Build. We have to pass `--inline-builds` to `yarn install`. However, that argument does not work for `yarn workspaces focus`.

**Comment by clemyan** [comment]
> It's never worked in Google Cloud Build

Looks like that will be fixed in the next release via #6377

Anyway, you can force CI to be detected by setting environment variable `CI='true'`. Alternatively, you can directly set the value of `enableInlineBuilds` via `yarn config set enableInlineBuilds true` or setting environment variable `YARN_ENABLE_INLINE_BUILDS='true'`. Both of those should work for all installs, including `yarn workspaces focus`.

Also, this popped into my notifications earlier (that's why I commented in the first place) but nothing happened to trigger that notification? Strange

**Comment by Robula** [comment]
@clemyan That's really great to know, we will try that. Thank you. 👍
